Tests/NetworkingTests/Mocks/FileWriterSpy.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  FileDataWriterSpy.swift
+//
+//
+//  Created by Jan Kodeš on 24.01.2024.
+//
+
+import Foundation
+import Networking
+
+class FileDataWriterSpy: FileDataWriterProtocol {
```

Comment:
Could you please include at least one sentence about the object for documentation? 🙏